backend/.env

```diff
@@ -1,7 +1,8 @@
 REST_API_PORT=5000
 API_DOCS_ENDPOINT_URL=/rest-api-docs
 
-MONGO_REPOSITORIES=DISABLED
+#MONGO_REPOSITORIES=DISABLED
+MONGO_REPOSITORIES=ENABLE
```

Comment:
Czy ten plik na pewno chcemy edytowac? Mozesz sobie zrobic plik .env.local i tam nadpisywac to co chcesz na swoje potrzeby.

Reply:
oooooo :)))))
Jak dodam taki plik z palca to wystarczy, czy może trzeba jeszcze gdzieś ścieżkę?
Dodać ten plik do git ignore?
Czy może Google pytać jak wrócę?

Reply:
Tak, do gitignore i z palca chyba powinno wystarczyc. A jzu w sumie jest taki plik widze :)

Reply:
ok